src/Application.php

```diff
@@ -640,7 +640,9 @@ protected function registerLogBindings()
      */
     protected function getMonologHandler()
     {
-        return (new StreamHandler(storage_path('logs/lumen.log'), Logger::DEBUG))
+        $logPath = env('APP_LOG_PATH', storage_path('logs/lumen.log'));
```

Comment:
I could add a config file. I do understand I can just extend the Application class, but I thought it can be convenient for others.